specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-05-01/frontdoor.json

```diff
@@ -0,0 +1,2348 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-05-01",
+    "title": "FrontDoorManagementClient",
+    "description": "Use these APIs to manage Azure Front Door resources through the Azure Resource Manager. You must make sure that requests made to these resources are secure."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Network/checkFrontDoorNameAvailability": {
+      "post": {
+        "tags": [
+          "CheckFrontDoorNameAvailability"
+        ],
+        "description": "Check the availability of a Front Door resource name.",
+        "operationId": "CheckFrontDoorNameAvailability",
```

Comment:
We use  "OperationGroup_Get" as the format for operationid.

Reply:
I looked into updating this, but adding `_Get` to these two routes wouldn't make sense since they're under the `post` verb, and adding a suffix like `_Update` or `_Create` wouldn't help either.